.circleci/config.yml

```diff
@@ -322,9 +322,6 @@ workflows:
                 - main
       - release-management/release-package:
           tag: latest-rc
-          pre-job-steps:
```

Comment:
This is a critical part of the federated plugin model - the agreement is that we'll pull in whatever is latest

That being said, I agree that we should test before bumping all of the dependencies. Is it possible to instead bump the dependencies before the `test-pack-tarballs` job?

Reply:
Just made some changes to keep bumping dependencies but also ensure we're testing them before the release actually happens.

Long term I think we need to re-think the `sfdx` release flow a bit more. Might need to make some changes to the orb's jobs and simplify this `.circleci.config.yml`

Reply:
Actually... what I tried won't work. I didn't realize `pre-job-steps` was just a parameter of the release job.

Reply:
I think we're going to have to make changes to the orb to really fix this properly. But for now this will make it so we don't accidentally release more untested dependencies.